sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -2978,6 +2978,8 @@ def aggregate(self, func, axis, *args, **kwargs):
   agg = aggregate
 
   applymap = frame_base._elementwise_method('applymap', base=pd.DataFrame)
```

Comment:
do we need to skip applymap for 2.1 around line 2980?

Reply:
So the thing is that it isn't removed in 2.1, it is just deprecated. Also, the replacement, `map`, isn't in Pandas until 2.1 so there isn't time to change over otherwise. We still want to include it until it is removed.